### DISCUSSION TITLE
Is there a way to only update certain fields in update()?

### DISCUSSION THREAD

**Discussion by aliakhtar** [collerek/ormar#141] [Q&A] [answered]
Is there an equivalent of `exclude_fields()` for `update()` to only update certain fields when update() is called?

I have multiple concurrent requests happening - in each request I only want to update one particular field - but because update() saves the whole model, its possible that there will be race conditions and one request will accidentally overwrite the changes of a previous request, even though they are updating different fields each.

**Comment by collerek** [comment, marked as answer]
Yeah, it's a clear omission. 
In `bulk_update` you have `columns: List` to set columns that should be updated.
In `QuerySet.update` (so `Model.objects.update(**kwargs)`) you only update params passed as `kwargs`.

And in 0.10.2 (PR #140) that will be merged this week I added `_columns: List[str]` to `Model.update()` where you can pass fields that should be updated (if omitted or empty all fields are updated)

**Reply by aliakhtar** [reply]
Thanks @collerek , when that new PR is merged, kindly shoot me a tag here please so I can update my dependencies? 🙏🏾

**Comment by collerek** [comment]
@aliakhtar Aaaaaand, it's out ;)
